### PR TITLE
initial BurstCube schema

### PR DESCRIPTION
**Pull request by jracusin**

### CODE REVIEW

File: gcn/notices/burstcube/Alert.example.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  "mission": "BurstCube",
+  "record_number": 1,
+  "trigger_time": "2024-06-01T00:00:00.0Z",
+  "event_name": ["GRB240601A"],
+  "id": ["b240601000"],
+  "ra": 232.0,
+  "dec": -53.1,
+  "uncertainty_shape": "circle",
+  "ra_uncertainty": [-4.6, 6.1],
+  "dec_uncertainty": [-3.4, 5.6],
```
Comment (Vidushi-GitHub) [inline gcn/notices/burstcube/Alert.example.json:15]: dec is not required for circle.

File: gcn/notices/burstcube/Alert.example.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  "systematic_included": false,
+  "healpix_url": "https://heasarc.gsfc.nasa.gov/burstcube/trigger/b240601000/current/b240601000_locmap.fits",
+  "healpix_file": "stuff",
+  "rate_snr": 7.2,
+  "rate_duration": 1.024,
+  "rate_energy_range": [50, 300],
+  "classification": { "GRB": 0.95, "SF": 0.03, "Local Particles": 0.02 },
+  "t90": 2.3,
+  "t90_error": [0.1, 0.1],
```
Comment (Vidushi-GitHub) [inline gcn/notices/burstcube/Alert.example.json:24]: We should also describe the -+ sign properly, is it already in definition, or one should provide proper sign.

**Comment by lpsinger** [comment]
@jracusin, did anyone review this code before you merged it?

**Comment by Vidushi-GitHub** [comment]
I reviewed first draft and later Dakota was helping a bit during AAS.

**Comment by lpsinger** [comment]
Thanks.

**Pull request merged**